CONTRIBUTING.rst

```diff
@@ -174,3 +174,22 @@ Then you are done.
 
 If the pull request has been opened for the ``f25-devel`` branch, then you also need to merge the ``f25-devel``
 branch back to ``master`` once you merge your pull request (see "Merging the Fedora devel branch back to the master branch" above).
+
+Pure community features
+-----------------------
+
+The pure community features are features which are part of the Anaconda code base but they are maintained and extended mainly by the community. These features are not a priority for the Anaconda project.
+
+In case of issues in pure community features, the Anaconda team will provide only sanity checking. It is the responsibility of the community (maintainers of the feature) to provide fix for the issue. If the issue will have bigger impact on other parts of the Anaconda project or if it will block a release or another priority feature and the fix won't be provided in a reasonable time the Anaconda team reserves the rights to remove or disable this feature from the Anaconda code base.
+
+Below is a list of pure community features, their community maintainers, and maintainers contact information:
+
+/boot on btrfs subvolume
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Origin: https://github.com/rhinstaller/anaconda/pull/1375
+- Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1418336
+- Maintainer:
```

Comment:
How is this defined? GitHub user? FAS ID? A person's name?

Reply:
I would not insist on one specific type of contact. Let's leave everyone to decide what contact is the best to reach that person. However, at least email should be always marked here as the base ground. We can always add more contacts if needed.

So for you, I will add a new commit with:
Maintainer: ``Neal Gompa <ngompa13@gmail.com>``

Do you want to add more contacts?

------------------------

About @npmcallum I will not add him here until his reply or PR to this document will arrive. Just to be sure that he really want to be a maintainer.

Reply:
That's fine with me.